docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -33,16 +33,16 @@ To override the default cache output behavior, pass an array of globs to a [`pip
 }
 ```
 
-If your task does not emit any files (e.g. unit tests with Jest), set `outputs` to an empty array (i.e. `[]`). Turborepo will automagically cache the logs for you.
+If your task does not emit any files (e.g. unit tests with Jest), set `outputs` to an empty array (i.e. `[]`), and Turborepo will cache the logs for you.
 
 <Callout type="info">
   Pro Tip for caching ESLint: You can get a cacheable pretty terminal output
   (even for non-errors) by setting `TIMING=1` variable before `eslint`. Learn
-  more over at the [ESLint
-  docs](https://eslint.org/docs/latest/developer-guide/working-with-rules#per-rule-performance)
+  more over in the [ESLint
+  docs](https://eslint.org/docs/latest/developer-guide/working-with-rules#per-rule-performance).
 </Callout>
 
-With the above pipeline configuration and proper output folders, if you were to run:
+With the pipeline configuration above and proper output folders, if you were to run:
```

Comment:
Normally, it's best to avoid the use of location. However, this structure is still awkward so maybe something like:

```suggestion
Once you have the pipeline configuration and output folders, you can run `turbo run build test` to create the cached artifacts. To confirm open up your `node_modules/.cache/turbo` to view the artifacts.
```

Reply:
good call. There are several places that do this that I'll have to defer at the moment, but I've updated this particular example